tests/nvme_test.py

```diff
@@ -277,12 +277,12 @@ def test_sanitize(self):
 
 
 class NVMeFabricsTestCase(NVMeTest):
-    HOSTNQN = 'libblockdev_nvmeof_hostnqn'
-    SUBNQN = 'libblockdev_nvmeof_subnqn'
+    SUBNQN = 'libblockdev_nvme'
 
     def setUp(self):
         self.loop_devs = []
         self.dev_files = []
+        self.hostnqn = get_nvme_hostnqn()
```

Comment:
If the hostnqn doesn't change between tests, we should move this to `setUpClass`.

Reply:
Good point. We might want to extend the tests with `hostnqn` changes in the future. This has been historically source of issues in `libnvme`. I'd leave this as is for the moment.